mitosheet/src/components/taskpanes/SplitTextToColumns/SplitTextToColumnsTaskpane.tsx

```diff
@@ -0,0 +1,302 @@
+import React, { useEffect, useState } from "react";
+import MitoAPI from "../../../jupyter/api";
+import { AnalysisData, ColumnID, GridState, SheetData, SplitTextToColumnsParams, StepType, UIState } from "../../../types"
+import DefaultEmptyTaskpane from "../DefaultTaskpane/DefaultEmptyTaskpane";
+import DefaultTaskpane from "../DefaultTaskpane/DefaultTaskpane";
+import DefaultTaskpaneBody from "../DefaultTaskpane/DefaultTaskpaneBody";
+import DefaultTaskpaneHeader from "../DefaultTaskpane/DefaultTaskpaneHeader";
+import Row from "../../spacing/Row";
+import Col from "../../spacing/Col";
+import Select from "../../elements/Select";
+import DropdownItem from "../../elements/DropdownItem";
+import { getDisplayColumnHeader } from "../../../utils/columnHeaders";
+import MultiSelectButtons from "../../elements/MultiSelectButtons";
+import useSendEditOnClick from "../../../hooks/useSendEditOnClick";
+import TextButton from "../../elements/TextButton";
+import Input from "../../elements/Input";
+import { useDebouncedEffect } from "../../../hooks/useDebouncedEffect";
+import '../../../../css/taskpanes/SplitTextToColumns/SplitTextToColumns.css'
+import DefaultTaskpaneFooter from "../DefaultTaskpane/DefaultTaskpaneFooter";
+
+interface SplitTextToColumnsTaskpaneProps {
+    mitoAPI: MitoAPI;
+    analysisData: AnalysisData;
+    sheetDataArray: SheetData[];
+    gridState: GridState;
+    setUIState: React.Dispatch<React.SetStateAction<UIState>>;
+    dfNames: string[];
+    startingColumnID?: ColumnID
+}
+
+type DelimiterObj = {
+    included: boolean,
+    delimiter: string
+}
+
+const defaultDelimitersObj: Record<string, DelimiterObj> = {
+    'Comma': {
+        included: false,
+        delimiter: ','
+    },
+    'Dash': {
+        included: false,
+        delimiter: '-'
+    },
+    'Tab': {
+        included: false,
+        delimiter: '\t'
+    },
+    'Space': {
+        included: false,
+        delimiter: ' '
+    },
+    'Other': {
+        included: false,
+        delimiter: ''
+    }
+}
+
+/* 
+    This taskpane allows users to split a column into multiple columns 
+    by separating on a delimeter
+*/
+const SplitTextToColumnsTaskpane = (props: SplitTextToColumnsTaskpaneProps): JSX.Element => {
+
+    // Use the selected column, unless its the index column, then just use the first column
+    const startingColumnIndex = props.gridState.selections[0].startingColumnIndex >= 0 ? props.gridState.selections[0].startingColumnIndex : 0
```

Comment:
Dont' think you want the first selection here, vs. the last one. But I'd check elsewhere in the codebase, we have a standard practice we do here when pulling a single column out of the selections.

Reply:
Ooo it actually looks like we don't have a standard. When setting the column formula, for example, we use the first column and when opening the column control panel, we use the second column. 

I'm going to try to unify it to the first column, since that is what Excel does.

Reply:
We now always default to the top left corner of the last selection.